src/devices/transformServer/FrameTransformServer.cpp

```diff
@@ -224,6 +224,99 @@ void FrameTransformServer::list_response(yarp::os::Bottle& out)
     }
 }
 
+string FrameTransformServer::get_matrix_as_text(Transforms_server_storage* storage, int i)
+{
+    //add here the latex to code to display the trnasformation matrix
+    //@@@TO BE COMPLETED
+    string s = "\\begin{ bmatrix } \
```

Comment:
Codacy found an issue: [Variable 's' is assigned a value that is never used.](https://app.codacy.com/manual/robotology/yarp/pullRequest?prid=5969044)